loopchain/channel/channel_service.py

```diff
@@ -496,23 +492,42 @@ def _handle_exception(future: asyncio.Future):
             )
 
         except Exception as e:
-            logging.warning(f"Due to Subscription fail to RadioStation(mother peer), "
+            logging.warning(f"Due to Subscription fail to RS peer({ChannelProperty().radio_station_target}), "
                             f"automatically retrying subscribe call")
+            return
 
         if response and response['response_code'] == message_code.Response.success:
-            logging.debug(f"Subscription to RadioStation(mother peer) is successful.")
+            logging.debug(f"Subscription to RS peer({ChannelProperty().radio_station_target}) is successful.")
             event.set()
             self.start_check_last_block_rs_timer()
 
-    def __check_last_block_to_rs(self):
-        last_block = self.__radio_station_stub.call_async("GetLastBlock")
-        if last_block['height'] <= self.__block_manager.get_blockchain().block_height:
+    def __unsubscribe_call_by_rest_stub(self):
+        if conf.REST_SSL_TYPE == conf.SSLAuthType.none:
```

Comment:
This conf is this node's option not rs's option.
So in my opinion it is dangerous to determine rs's option based on my option.